clients/venice-thin-client/src/test/java/com/linkedin/venice/client/schema/RouterBackedSchemaReaderTest.java

```diff
@@ -226,6 +228,51 @@ public void testGetLatestValueSchema()
     Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(1)).getRaw(Mockito.anyString());
   }
 
+  @Test
+  public void testGetLargestValueSchemaId()
```

Comment:
Let's rename testGetLargestValueSchemaId -> testGetMaxValueSchemaId to align with actual method name.